app/actions/AccountMixerActions.js

```diff
@@ -144,6 +145,7 @@ export const stopAccountMixer = (cleanLogs) => {
     dispatch({ type: STOPMIXER_ATTEMPT });
     try {
       mixerStreamer.cancel();
+      wallet.lockWallet(walletService);
```

Comment:
```suggestion
      wallet.lockWallet(sel.walletService(getState()));
```

You can call it directly